adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1562,11 +1562,29 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
     stringency: ValidationStringency = ValidationStringency.STRICT)(implicit s: DummyImplicit): AlignmentRecordRDD = LoadIndexedBam.time {
 
     val path = new Path(pathName)
+
+    // If pathName is a directory of bam files, scan all files to
+    // select index files.
+    val indexPath = if (pathName.endsWith(".bam"))
+      new Path(pathName + ".bai")
+    else
+      path
+
     // todo: can this method handle SAM and CRAM, or just BAM?
```

Comment:
Can you update this comment to `// currently only supports BAM` and cross reference https://github.com/bigdatagenomics/adam/issues/1833.